packages/web/components/complex/pools-table.tsx

```
@@ -500,11 +500,11 @@ export const PoolsTable = (props: PropsWithChildren<PoolsTableProps>) => {
   );
 };
 
-export type PoolCellComponent<TProps = {}> = FunctionComponent<
+type PoolCellComponent<TProps = {}> = FunctionComponent<
```

Biome

[error] Don't use '{}' as a type.

Prefer explicitly define the object shape. '{}' means "any non-nullable value".

(lint/complexity/noBannedTypes)